test/e2e/framework/volume_util.go

```diff
@@ -174,21 +179,31 @@ func NewGlusterfsServer(cs clientset.Interface, namespace string) (config Volume
 	return config, pod, ip
 }
 
-// iSCSI-specific wrapper for CreateStorageServer.
-func NewISCSIServer(cs clientset.Interface, namespace string) (config VolumeTestConfig, pod *v1.Pod, ip string) {
+// iSCSI-specific wrapper for CreateStorageServer. Note that there can be only
```

Comment:
Now we are fixing golint failures under `test/e2e/framework`.
So please add "NewISCSIServer is an" for avoiding golint failures.

Reply:
Fixed